rvs/conf/deviceid.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
When this .sh file will be called?

Reply:
This file is called from testscripts/*.sh files. So after build if the user calls any file under testscripts which is copied to build/bin ex gst.new.sh/pesm.new.sh then this .sh file get called.